src/visualization/utils/annotationUtils.ts

```diff
@@ -199,6 +202,49 @@ export const makeAnnotationLayer = (
   return null
 }
 
+export const addAnnotationLayer = (
+  config: Config,
+  inAnnotationWriteMode: boolean,
+  cellID: string,
+  xColumn: string,
+  yColumn: string,
+  groupKey: string[],
+  annotations: AnnotationsList,
+  annotationsAreVisible: boolean,
+  dispatch: Dispatch<any>,
+  eventPrefix = 'xyplot'
+) => {
+  if (isFlagEnabled('annotations')) {
```

Comment:
what do you think of inverting this to a guard statement like:

```ts
if (!isFlagEnabled('annotations')) {
  return
}
```

that way, we get the body of the function at the highest indentation level

Reply:
sure, why not.